Packs/GoogleChronicleBackstory/Integrations/GoogleChronicleBackstory/GoogleChronicleBackstory.py

```diff
@@ -1311,14 +1446,66 @@ def gcb_list_alerts_command(client_obj, args: Dict[str, Any]):
         hr += 'No Records Found'
         return hr, {}, {}
 
+    # prepare alerts into human readable
+    hr = convert_alerts_into_hr(alerts)
+
+    # Remove Url key in context data
+    for alert in alerts:
+        for alert_info in alert.get('AlertInfo', []):
+            if 'Uri' in alert_info.keys():
+                del alert_info['Uri']
     ec = {
         CHRONICLE_OUTPUT_PATHS['Alert']: alerts
     }
 
+    return hr, ec, alerts
+
+
+def gcb_list_events_command(client_obj, args: Dict[str, str]):
+    """
+    List all of the events discovered within your enterprise on a particular device within the specified time range.
+
+    :type client_obj: Client
+    :param client_obj: client object which is used to get response from api
+
+    :type args:  Dict[str, str]
+    :param args: it contain arguments of gcb-list-ioc command
+
+    :return: command output
+    :rtype: str, dict, dict
+    """
+
+    asset_identifier_type = ASSET_IDENTIFIER_NAME_DICT.get(args.get('asset_identifier_type', '').lower(),
+                                                           args.get('asset_identifier_type', ''))
```

Comment:
In case of predefined selection for `asset_identifier_type `we have handled case sensitive in case it is written in pascal or any format, we will always convert into lower case and then pass the assigned key to the argument. But if supplied `asset_identifier_type ` is not a part of selection then we should be passing as it is.
so yes, this should be as it is.